Parser/pegen/pegen.c

```diff
@@ -38,6 +38,80 @@ byte_offset_to_character_offset(PyObject *line, int col_offset)
     return size;
 }
 
+const char *
+get_expr_name(expr_ty e)
+{
```

Comment:
Why not just make the function in ast.c public? This seems just silly code replication.

Reply:
I though about it as well and I figured it would make the transition to pegen being the only parser easier, if we don't have to copy code from ast.c, before we delete it.

Reply:
Okay then.